libs/design/src/atoms/form/form-field/form-field/form-field.md

```diff
@@ -9,4 +9,29 @@
   <input daff-input type="text" placeholder="Email" name="email" />
   <daff-error-message *ngIf="requiredError">This field is required.</daff-error-message>
 </daff-form-field>
-```
\ No newline at end of file
+```
+
+## Creating a DaffFormFieldControl that works with DaffFormField
+
+Creating a control that works easily with the form field is fairly straightforward. We've provided the `DaffAbstractFormControl` (abstract class/interface) to allow you to implement the requirements on your control in a consistent manner. This will also ensure that any breaking updates will be caught at build time, instead of runtime.
```

Comment:
`DaffAbstractFormControl` should be `DaffFormFieldControl`, right?

Reply:
@lderrickable resolved. 74d7ebb3d2e3cfe08c03b5ff3e797936c29fcb9c